src/app/public/modules/definition-list/types/definition-list-mode.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Specifies the mode for the definition list.
+ */
+export type SkyDefinitionListMode = 'default' | 'nameValuePair' | 'termDescription';
```

Comment:
Instead of "default" could we find a name that's more descriptive of the current, default behavior?

You could potentially convert this to a string `enum` so that documenting each mode is easier.
```
export string enum SkyDefinitionListMode {
  /**
   * This mode blah blah blah
   */
  NameValuePair = 'nameValuePair'
}
```

Reply:
I'm not sure what the default behavior is, so I'm not sure what descriptive term to use, but let me know if you want some help with the terminology.

Also the current description users `mode` to define the `mode` property, which seems less than ideal. I'd update this description to something more like `Specifies the type of label-value pair to display in the definition list.` (Let me know if you want to fine-tune this further to be more descriptive. I can't really tell what this property does. I tried to swap between default, nameValuePair, and termDescription in Stackblitz, but I don't see any visible difference between the three.)

Reply:
Oh, and should we update the demo so that users can swap between the different modes to see them? And also add an option to swap the orientation? Both of those strike me as more valuable options for the demo than the current option to hide the header (especially if the header is going out the door in the next release).

Reply:
Great point about documentation. I've made changes based on your feedback.

Reply:
Demo has been updated.